content/de/resources/tutorials/overview.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Tutorials and Guides
+description: Tutorials and Guides
+category: 'Tutorials'
+categoryIcon: 'mdi-school'
+position: 1
+---
+
+<v-row>
+  <dwf-tutorials></dwf-tutorials>
```

Comment:
Die Karte wird nicht angezeigt. Du müsstest dafür in der Datei _.vue die Komponente importieren und registrieren.

Reply:
Das ist komisch. Ich hab's an 2 unterschiedlichen PCs ausprobiert und sehe jedes mal das Tutorial

Reply:
Hab den Fehler gefunden. Ich hatte einen `/` zu viel. Im dev Modus ist das nicht aufgefallen, aber wenn ich den Code generieren lass und die html Dateien über einen Webserver aufrufe dann geht's nicht.

Danke für den Tipp!